apps/studio/src/components/CmsSidebar/CmsSideNav.tsx

```diff
@@ -137,12 +139,8 @@ const SideNavItem = ({
   const siteProps = useQueryParse(siteSchema)
   const isCurResourceActive = getCurResource(siteProps) === resourceId
 
-  if (isLoading || !data) {
-    return (
-      <Flex align="center" height="2rem" pl="2.75rem" pr="1rem">
-        <Spinner />
-      </Flex>
-    )
+  if (!data) {
```

Comment:
this will always be true even if loading, meaning the skeleton below wrapping the component is useless. Might be easier to stick to the old conditional

```suggestion
  if (isLoading || !data) {
```

Reply:
done! didn't check `width` on children cos the load takes forever, so might be the case where, for example, third child is doubly indented (arrow for illustration), but the fourth child (ie child of third child) is full size skeleton while loading: 

```
>> <child />
<Skeleton />
```